### PR TITLE
License reference document

### PR DESCRIPTION
**Pull request by badhrinathpa**
Adding Reference document to list references of code taken from repos which are license comaptible.

### CODE REVIEW

**Comment by krsna1729** [comment]
@badhrinathpa DPDK largely BSD-3 http://git.dpdk.org/dpdk/tree/license no apache-2

File: REFERENCE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+****************************************************************************************************
+All OPENAIRINTERFACE References are listed below and are covered under Apache 2.0 License.
```
Comment (thakurajayL) [inline REFERENCE]: Would be good if add all documentation related files in the markdown format.

Comment (thakurajayL) [reply REFERENCE]: # LICENSE INFORMATION
>This page should capture all third party software used in the project **OMEC** and list the license information of those components. Even if full information about the license is not available its always good to list the third party library, component, source code here.
>
## OPENAIRINTERFACE License  

- openinterface is covered under APACHE 2.0 license
-- https://github.com/OPENAIRINTERFACE/openair-cn/tree/master/src/hss_rel14/hsssec : c3po/hsssec
-- https://github.com/OPENAIRINTERFACE/openair-cn/tree/master/src/hss_rel14/util   : c3po/util

## DPDK License
- DPDK Reference is covered under Apache 2.0 License.

## SSL License 
- Reference to SSL is covered under Apache 2.0 License. 
-- https://tls.mbed.org/ : c3po/sgxcdr/dealer/Enclave

## Eclipse License
- Reference to Eclipse SGX is covered under EPL 1.0 License.
-- https://01.org/intel-softwareguard-extensions : c3po/sgxcdr/kms/App
-- https://01.org/intel-softwareguard-extensions : c3po/sgxcdr/dealer/App

Comment (thakurajayL) [reply REFERENCE]: Use this file in raw format with .md extension. Also update content if something is wrong.

**Comment by krsna1729** [comment]
Add hyperlinks to license files. I could not find where SGX was listed Eclipse license. SGX driver is BSD-3/GPLv2 and SGX SDK is BSD-3. So not sure where you are getting this info from.

**Comment by badhrinathpa** [comment]
Review comments addressed. Request to please review and approve.

**Pull request merged**